paasta_tools/vitesscluster_tools.py

```diff
@@ -48,8 +564,102 @@ def __init__(
 
-    def get_instances(self, with_limit: bool = True) -> int:
-        return self.config_dict.get("replicas", 1)
+    def get_namespace(self) -> str:
+        return self.config_dict.get("namespace", KUBERNETES_NAMESPACE)
+
+    def get_images(self) -> Dict[str, str]:
+        vitess_images = self.config_dict.get(
+            "images", load_system_paasta_config().get_vitess_images()
+        )
+        return {
+            "vtctld": vitess_images["vtctld_image"],
+            "vtadmin": vitess_images["vtadmin_image"],
+            "vtgate": vitess_images["vtgate_image"],
+            "vttablet": vitess_images["vttablet_image"],
+        }
+
+    def get_globalLockserver(self) -> Dict[str, str]:
+        zk_address = self.config_dict.get("zk_address")
+        return {
+            "external": {
+                "implementation": TOPO_IMPLEMENTATION,
+                "address": zk_address,
+                "rootPath": TOPO_GLOBAL_ROOT,
+            }
+        }
+
+    def get_cells(self) -> List[str]:
+        paasta_pool = self.get_pool()
+        paasta_cluster = self.config_dict.get("paasta_cluster", self.get_cluster())
+        cells = self.config_dict.get("cells")
+        region = self.config_dict.get("region")
+        vtgate_resources = self.config_dict.get("vtgate_resources")
+        return [
+            get_cell_config(cell, paasta_pool, paasta_cluster, region, vtgate_resources)
+            for cell in cells
+        ]
+
+    def get_vitessDashboard(self) -> Dict[str, str]:
+        paasta_pool = self.get_pool()
+        paasta_cluster = self.config_dict.get("paasta_cluster", self.get_cluster())
+        cells = self.config_dict.get("cells")
+        zk_address = self.config_dict.get("zk_address")
+        vtctld_resources = self.config_dict.get("vtctld_resources")
+        return get_vitess_dashboard_config(
+            cells, paasta_pool, paasta_cluster, zk_address, vtctld_resources
+        )
+
+    def get_vtadmin(self) -> Dict[str, str]:
+        paasta_pool = self.get_pool()
+        paasta_cluster = self.config_dict.get("paasta_cluster", self.get_cluster())
+        cells = self.config_dict.get("cells")
+        vtadmin_resources = self.config_dict.get("vtadmin_resources")
+        return get_vt_admin_config(
+            cells, paasta_pool, paasta_cluster, vtadmin_resources
+        )
+
+    def get_keyspaces(self) -> List[Dict[str, str]]:
+        paasta_pool = self.get_pool()
+        paasta_cluster = self.config_dict.get("paasta_cluster", self.get_cluster())
+        cells = self.config_dict.get("cells")
+        zk_address = self.config_dict.get("zk_address")
+        region = self.config_dict.get("region")
+        keyspaces = self.config_dict.get("keyspaces")
+        return get_keyspaces_config(
+            cells, keyspaces, paasta_pool, paasta_cluster, zk_address, region
+        )
+
+    def get_updateStrategy(self) -> Dict[str, str]:
+        return self.config_dict.get("updateStrategy", {"type": "Immediate"})
```

Comment:
```suggestion
    def get_update_strategy(self) -> Dict[str, str]:
        return self.config_dict.get("update_strategy", {"type": "Immediate"})
```
(this will also require updating the key in our config dict)

Reply:
https://github.com/Yelp/paasta/pull/3889/commits/48ebe609e02377336248dcbb9bfed29a0d6213ae